### PR TITLE
feature/85_adding_to_run

### PR DESCRIPTION
**Pull request by Reagen**
See issue #85

### CODE REVIEW

File: analysis/deep2_r23_o32/run_general_script.py [hunk 1]
```diff
@@ -1,4 +1,23 @@
...
+general.run_grid_plots(fitspath_ini, dataset, raw=True, apply_dust=False,
+                       revised=True, individual=False)
+general.run_grid_plots(fitspath_ini, dataset, raw=False, apply_dust=True,
+                       revised=False, individual=False)
+general.run_grid_plots(fitspath_ini, dataset, raw=False, apply_dust=True,
+                       revised=True, individual=False)
+general.run_grid_plots(fitspath_ini, dataset, raw=True, apply_dust=True,
+                       revised=False, individual=False)
+general.run_grid_plots(fitspath_ini, dataset, raw=True, apply_dust=True,
+                       revised=True, individual=False)
```
Comment (astrochun) [inline analysis/deep2_r23_o32/run_general_script.py:23]: You can simplify this using a `dict` approach. Take a look at [this](https://github.com/astrochun/Metallicity_Stack_Commons/blob/master/tests/test_error_prop.py#L10-L38)

File: analysis/deep2_r23_o32/general.py [hunk 2]
```diff
@@ -29,7 +29,8 @@ def run_grid_r23_o32_analysis(dataset, n_split=3, y_correction=False,
            Version for spectrum in zoom_and_gauss_general. Default: False
     :param adaptive: bool. Set to use adaptive binning, otherwise equal
            log(R23) bins. Default: True
-    :param apply_dust: bool. Apply dust correction. Default: False
+    :param apply_dust: bool. Apply dust correction. Default: True
+                    By setting apply_dust=true, all cases are run
```
Comment (astrochun) [inline analysis/deep2_r23_o32/general.py:33]: Wouldn't it make more sense to still keep the default as `False` and then do: `general.run_grid_r23_o32_analysis(dataset, apply_dust=True)`

**Review by astrochun — COMMENTED** [review]
I still need to think about `run_grid_r23_o32_analysis` doing all the run with `apply_dust=True`. The issue here is that it will overwrite previous files (the non-dust-corrected ones).

File: analysis/deep2_r23_o32/run_general_script.py (hunk 1 above)
Comment (Reagen) [reply analysis/deep2_r23_o32/run_general_script.py:23]: fixed

**Review by astrochun — APPROVED** [review]
Changes look good. Ready to merge.

**Pull request merged**